feat(components): [button-group]: add direction prop

VisualYuki opened the pull request:
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

VisualYuki committed:
feat(components): [button-group]:add vertical direction for button group

VisualYuki commented:
[example](https://element-plus.run/#eyJBcHAudnVlIjoiPHNjcmlwdCBzZXR1cCBsYW5nPVwidHNcIj5cbmltcG9ydCB7IEVsZW1lbnRQbHVzIH0gZnJvbSAnQGVsZW1lbnQtcGx1cy9pY29ucy12dWUnXG5pbXBvcnQgeyB2ZXJzaW9uIGFzIGVwVmVyc2lvbiB9IGZyb20gJ2VsZW1lbnQtcGx1cydcbmltcG9ydCB7IHJlZiwgdmVyc2lvbiBhcyB2dWVWZXJzaW9uIH0gZnJvbSAndnVlJ1xuXG5jb25zdCBtc2cgPSByZWYoJ0hlbGxvIFdvcmxkIScpXG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8ZWwtYnV0dG9uLWdyb3VwPlxuICAgIDxlbC1idXR0b24gdHlwZT1cInByaW1hcnlcIj5idXR0b24xPC9lbC1idXR0b24+XG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwicHJpbWFyeVwiPmJ1dHRvbjI8L2VsLWJ1dHRvbj5cbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJwcmltYXJ5XCI+YnV0dG9uMyAxMjEyPC9lbC1idXR0b24+XG4gIDwvZWwtYnV0dG9uLWdyb3VwPlxuICA8c3Bhbj5fX188L3NwYW4+XG5cbiAgPGVsLWJ1dHRvbi1ncm91cD5cbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJwcmltYXJ5XCI+YnV0dG9uMTwvZWwtYnV0dG9uPlxuICAgIDxlbC1idXR0b24gdHlwZT1cInByaW1hcnlcIj5idXR0b24yPC9lbC1idXR0b24+XG4gIDwvZWwtYnV0dG9uLWdyb3VwPlxuICA8c3Bhbj5fX188L3NwYW4+XG4gIDxlbC1idXR0b24tZ3JvdXA+XG4gICAgPGVsLWJ1dHRvbj5idXR0b24xPC9lbC1idXR0b24+XG4gICAgPGVsLWJ1dHRvbj5idXR0b24yPC9lbC1idXR0b24+XG4gICAgPGVsLWJ1dHRvbj5idXR0b24zIDEyMTI8L2VsLWJ1dHRvbj5cbiAgPC9lbC1idXR0b24tZ3JvdXA+XG5cbiAgPHNwYW4+X19fPC9zcGFuPlxuXG4gIDxlbC1idXR0b24tZ3JvdXA+XG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwicHJpbWFyeVwiIHJvdW5kPnJvdW5kPC9lbC1idXR0b24+XG4gIDwvZWwtYnV0dG9uLWdyb3VwPlxuICA8c3Bhbj5fX188L3NwYW4+XG4gIDxlbC1idXR0b24tZ3JvdXA+XG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwicHJpbWFyeVwiIGNpcmNsZT5jaXJjbGU8L2VsLWJ1dHRvbj5cbiAgPC9lbC1idXR0b24tZ3JvdXA+XG4gIDxzcGFuPl9fXzwvc3Bhbj5cbiAgPGVsLWJ1dHRvbi1ncm91cD5cbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJwcmltYXJ5XCI+cHJpbWFyeTwvZWwtYnV0dG9uPlxuICA8L2VsLWJ1dHRvbi1ncm91cD5cbiAgPHNwYW4+X19fPC9zcGFuPlxuICA8ZWwtYnV0dG9uLWdyb3VwPlxuICAgIDxlbC1idXR0b24gdHlwZT1cInByaW1hcnlcIj5idXR0b24zIDEyMTI8L2VsLWJ1dHRvbj5cblxuICAgIDxlbC1kcm9wZG93bj5cbiAgICAgIDxlbC1idXR0b24gdHlwZT1cInByaW1hcnlcIj5cbiAgICAgICAgRHJvcGRvd24gTGlzdDxlbC1pY29uIGNsYXNzPVwiZWwtaWNvbi0tcmlnaHRcIj48YXJyb3ctZG93biAvPjwvZWwtaWNvbj5cbiAgICAgIDwvZWwtYnV0dG9uPlxuICAgICAgPHRlbXBsYXRlICNkcm9wZG93bj5cbiAgICAgICAgPGVsLWRyb3Bkb3duLW1lbnU+XG4gICAgICAgICAgPGVsLWRyb3Bkb3duLWl0ZW0+QWN0aW9uIDE8L2VsLWRyb3Bkb3duLWl0ZW0+XG4gICAgICAgICAgPGVsLWRyb3Bkb3duLWl0ZW0+QWN0aW9uIDI8L2VsLWRyb3Bkb3duLWl0ZW0+XG4gICAgICAgICAgPGVsLWRyb3Bkb3duLWl0ZW0+QWN0aW9uIDM8L2VsLWRyb3Bkb3duLWl0ZW0+XG4gICAgICAgICAgPGVsLWRyb3Bkb3duLWl0ZW0+QWN0aW9uIDQ8L2VsLWRyb3Bkb3duLWl0ZW0+XG4gICAgICAgICAgPGVsLWRyb3Bkb3duLWl0ZW0+QWN0aW9uIDU8L2VsLWRyb3Bkb3duLWl0ZW0+XG4gICAgICAgIDwvZWwtZHJvcGRvd24tbWVudT5cbiAgICAgIDwvdGVtcGxhdGU+XG4gICAgPC9lbC1kcm9wZG93bj5cbiAgPC9lbC1idXR0b24tZ3JvdXA+XG5cbiAgPGJyIC8+XG4gIDxiciAvPlxuXG4gIDxlbC1idXR0b24tZ3JvdXAgOnZlcnRpY2FsPVwidHJ1ZVwiPlxuICAgIDxlbC1idXR0b24+YnV0dG9uMTwvZWwtYnV0dG9uPlxuICAgIDxlbC1idXR0b24+YnV0dG9uMiAxPC9lbC1idXR0b24+XG4gICAgPGVsLWJ1dHRvbj5idXR0b24zIDEyMTI8L2VsLWJ1dHRvbj5cbiAgPC9lbC1idXR0b24tZ3JvdXA+XG4gIDxzcGFuPl9fXzwvc3Bhbj5cbiAgPGVsLWJ1dHRvbi1ncm91cCA6dmVydGljYWw9XCJ0cnVlXCI+XG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwicHJpbWFyeVwiPmJ1dHRvbjE8L2VsLWJ1dHRvbj5cbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJwcmltYXJ5XCI+YnV0dG9uMjwvZWwtYnV0dG9uPlxuICAgIDxlbC1idXR0b24gdHlwZT1cInByaW1hcnlcIj5idXR0b24zIDEyMTI8L2VsLWJ1dHRvbj5cbiAgPC9lbC1idXR0b24tZ3JvdXA+XG4gIDxzcGFuPl9fXzwvc3Bhbj5cbiAgPGVsLWJ1dHRvbi1ncm91cCA6dmVydGljYWw9XCJ0cnVlXCI+XG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwicHJpbWFyeVwiIHJvdW5kPnJvdW5kPC9lbC1idXR0b24+XG4gIDwvZWwtYnV0dG9uLWdyb3VwPlxuICA8c3Bhbj5fX188L3NwYW4+XG4gIDxlbC1idXR0b24tZ3JvdXAgOnZlcnRpY2FsPVwidHJ1ZVwiPlxuICAgIDxlbC1idXR0b24gdHlwZT1cInByaW1hcnlcIiBjaXJjbGU+Y2lyY2xlPC9lbC1idXR0b24+XG4gIDwvZWwtYnV0dG9uLWdyb3VwPlxuICA8c3Bhbj5fX188L3NwYW4+XG4gIDxlbC1idXR0b24tZ3JvdXAgOnZlcnRpY2FsPVwidHJ1ZVwiPlxuICAgIDxlbC1idXR0b24gdHlwZT1cInByaW1hcnlcIj5wcmltYXJ5PC9lbC1idXR0b24+XG4gIDwvZWwtYnV0dG9uLWdyb3VwPlxuICA8c3Bhbj5fX188L3NwYW4+XG4gIDxlbC1idXR0b24tZ3JvdXAgOnZlcnRpY2FsPVwidHJ1ZVwiPlxuICAgIDxlbC1idXR0b24gdHlwZT1cInByaW1hcnlcIj5idXR0b24xPC9lbC1idXR0b24+XG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwicHJpbWFyeVwiPmJ1dHRvbjI8L2VsLWJ1dHRvbj5cbiAgPC9lbC1idXR0b24tZ3JvdXA+XG5cbiAgPHNwYW4+X19fPC9zcGFuPlxuICA8ZWwtYnV0dG9uLWdyb3VwIDp2ZXJ0aWNhbD1cInRydWVcIj5cbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJwcmltYXJ5XCI+YnV0dG9uMTwvZWwtYnV0dG9uPlxuICAgIDxlbC1kcm9wZG93bj5cbiAgICAgIDxlbC1idXR0b24gdHlwZT1cInByaW1hcnlcIj5cbiAgICAgICAgRHJvcGRvd24gTGlzdDxlbC1pY29uIGNsYXNzPVwiZWwtaWNvbi0tcmlnaHRcIj48YXJyb3ctZG93biAvPjwvZWwtaWNvbj5cbiAgICAgIDwvZWwtYnV0dG9uPlxuICAgICAgPHRlbXBsYXRlICNkcm9wZG93bj5cbiAgICAgICAgPGVsLWRyb3Bkb3duLW1lbnU+XG4gICAgICAgICAgPGVsLWRyb3Bkb3duLWl0ZW0+QWN0aW9uIDE8L2VsLWRyb3Bkb3duLWl0ZW0+XG4gICAgICAgICAgPGVsLWRyb3Bkb3duLWl0ZW0+QWN0aW9uIDI8L2VsLWRyb3Bkb3duLWl0ZW0+XG4gICAgICAgICAgPGVsLWRyb3Bkb3duLWl0ZW0+QWN0aW9uIDM8L2VsLWRyb3Bkb3duLWl0ZW0+XG4gICAgICAgICAgPGVsLWRyb3Bkb3duLWl0ZW0+QWN0aW9uIDQ8L2VsLWRyb3Bkb3duLWl0ZW0+XG4gICAgICAgICAgPGVsLWRyb3Bkb3duLWl0ZW0+QWN0aW9uIDU8L2VsLWRyb3Bkb3duLWl0ZW0+XG4gICAgICAgIDwvZWwtZHJvcGRvd24tbWVudT5cbiAgICAgIDwvdGVtcGxhdGU+XG4gICAgPC9lbC1kcm9wZG93bj5cbiAgPC9lbC1idXR0b24tZ3JvdXA+XG4gIDxiciAvPlxuICA8YnIgLz5cbjwvdGVtcGxhdGU+IiwiZWxlbWVudC1wbHVzLmpzIjoiaW1wb3J0IEVsZW1lbnRQbHVzIGZyb20gJ2VsZW1lbnQtcGx1cydcbmltcG9ydCB7IGdldEN1cnJlbnRJbnN0YW5jZSB9IGZyb20gJ3Z1ZSdcblxubGV0IGluc3RhbGxlZCA9IGZhbHNlXG5hd2FpdCBsb2FkU3R5bGUoKVxuXG5leHBvcnQgZnVuY3Rpb24gc2V0dXBFbGVtZW50UGx1cygpIHtcbiAgaWYgKGluc3RhbGxlZCkgcmV0dXJuXG4gIGNvbnN0IGluc3RhbmNlID0gZ2V0Q3VycmVudEluc3RhbmNlKClcbiAgaW5zdGFuY2UuYXBwQ29udGV4dC5hcHAudXNlKEVsZW1lbnRQbHVzKVxuICBpbnN0YWxsZWQgPSB0cnVlXG59XG5cbmV4cG9ydCBmdW5jdGlvbiBsb2FkU3R5bGUoKSB7XG4gIGNvbnN0IHN0eWxlcyA9IFsnaHR0cHM6Ly9wcmV2aWV3LTE4OTA2LWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguY3NzJywgJ2h0dHBzOi8vcHJldmlldy0xODkwNi1lbGVtZW50LXBsdXMuc3VyZ2Uuc2gvYnVuZGxlL2luZGV4LmNzcyddLm1hcCgoc3R5bGUpID0+IHtcbiAgICByZXR1cm4gbmV3IFByb21pc2UoKHJlc29sdmUsIHJlamVjdCkgPT4ge1xuICAgICAgY29uc3QgbGluayA9IGRvY3VtZW50LmNyZWF0ZUVsZW1lbnQoJ2xpbmsnKVxuICAgICAgbGluay5yZWwgPSAnc3R5bGVzaGVldCdcbiAgICAgIGxpbmsuaHJlZiA9IHN0eWxlXG4gICAgICBsaW5rLmFkZEV2ZW50TGlzdGVuZXIoJ2xvYWQnLCByZXNvbHZlKVxuICAgICAgbGluay5hZGRFdmVudExpc3RlbmVyKCdlcnJvcicsIHJlamVjdClcbiAgICAgIGRvY3VtZW50LmJvZHkuYXBwZW5kKGxpbmspXG4gICAgfSlcbiAgfSlcbiAgcmV0dXJuIFByb21pc2UuYWxsU2V0dGxlZChzdHlsZXMpXG59XG4iLCJ0c2NvbmZpZy5qc29uIjoie1xuICBcImNvbXBpbGVyT3B0aW9uc1wiOiB7XG4gICAgXCJ0YXJnZXRcIjogXCJFU05leHRcIixcbiAgICBcImpzeFwiOiBcInByZXNlcnZlXCIsXG4gICAgXCJtb2R1bGVcIjogXCJFU05leHRcIixcbiAgICBcIm1vZHVsZVJlc29sdXRpb25cIjogXCJCdW5kbGVyXCIsXG4gICAgXCJ0eXBlc1wiOiBbXCJlbGVtZW50LXBsdXMvZ2xvYmFsLmQudHNcIl0sXG4gICAgXCJhbGxvd0ltcG9ydGluZ1RzRXh0ZW5zaW9uc1wiOiB0cnVlLFxuICAgIFwiYWxsb3dKc1wiOiB0cnVlLFxuICAgIFwiY2hlY2tKc1wiOiB0cnVlXG4gIH0sXG4gIFwidnVlQ29tcGlsZXJPcHRpb25zXCI6IHtcbiAgICBcInRhcmdldFwiOiAzLjNcbiAgfVxufVxuIiwiUGxheWdyb3VuZE1haW4udnVlIjoiPHNjcmlwdCBzZXR1cD5cbmltcG9ydCBBcHAgZnJvbSAnLi9BcHAudnVlJ1xuaW1wb3J0IHsgc2V0dXBFbGVtZW50UGx1cyB9IGZyb20gJy4vZWxlbWVudC1wbHVzLmpzJ1xuc2V0dXBFbGVtZW50UGx1cygpXG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8QXBwIC8+XG48L3RlbXBsYXRlPlxuIiwiaW1wb3J0LW1hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge1xuICAgIFwidnVlXCI6IFwiaHR0cHM6Ly9jZG4uanNkZWxpdnIubmV0L25wbS9AdnVlL3J1bnRpbWUtZG9tQGxhdGVzdC9kaXN0L3J1bnRpbWUtZG9tLmVzbS1icm93c2VyLmpzXCIsXG4gICAgXCJAdnVlL3NoYXJlZFwiOiBcImh0dHBzOi8vY2RuLmpzZGVsaXZyLm5ldC9ucG0vQHZ1ZS9zaGFyZWRAbGF0ZXN0L2Rpc3Qvc2hhcmVkLmVzbS1idW5kbGVyLmpzXCIsXG4gICAgXCJlbGVtZW50LXBsdXNcIjogXCJodHRwczovL3ByZXZpZXctMTg5MDYtZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5mdWxsLm1pbi5tanNcIixcbiAgICBcImVsZW1lbnQtcGx1cy9cIjogXCJ1bnN1cHBvcnRlZFwiLFxuICAgIFwiQGVsZW1lbnQtcGx1cy9pY29ucy12dWVcIjogXCJodHRwczovL2Nkbi5qc2RlbGl2ci5uZXQvbnBtL0BlbGVtZW50LXBsdXMvaWNvbnMtdnVlQDIvZGlzdC9pbmRleC5taW4uanNcIlxuICB9LFxuICBcInNjb3Blc1wiOiB7fVxufSIsIl9vIjp7InNob3dIaWRkZW4iOnRydWUsInN0eWxlU291cmNlIjoiaHR0cHM6Ly9wcmV2aWV3LTE4OTA2LWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguY3NzIn19)

thinkasany commented on docs/examples/button/group.vue:
```diff
@@ -11,6 +11,12 @@
     <el-button type="primary" :icon="Delete" />
   </el-button-group>
+
+  <el-button-group class="ml-4" :vertical="true">
```
It seems that direction is mostly used in element components to display arrangement directions.
![image](https://github.com/user-attachments/assets/89f07061-280a-4594-9e5c-01896945d2e2)

thinkasany replied:
The usage here seems to be uncorrected.

VisualYuki committed:
feat(components): [button-group]: add direction prop (update)

VisualYuki commented:
@thinkasany  [update example](https://element-plus.run/#eyJBcHAudnVlIjoiPHNjcmlwdCBzZXR1cCBsYW5nPVwidHNcIj5cbmltcG9ydCB7IEVsZW1lbnRQbHVzLCBBcnJvd0Rvd24gfSBmcm9tICdAZWxlbWVudC1wbHVzL2ljb25zLXZ1ZSdcbmltcG9ydCB7IHZlcnNpb24gYXMgZXBWZXJzaW9uIH0gZnJvbSAnZWxlbWVudC1wbHVzJ1xuaW1wb3J0IHsgcmVmLCB2ZXJzaW9uIGFzIHZ1ZVZlcnNpb24gfSBmcm9tICd2dWUnXG5cbmNvbnN0IG1zZyA9IHJlZignSGVsbG8gV29ybGQhJylcbjwvc2NyaXB0PlxuXG48dGVtcGxhdGU+XG4gIDxlbC1idXR0b24tZ3JvdXA+XG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwicHJpbWFyeVwiPmJ1dHRvbjE8L2VsLWJ1dHRvbj5cbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJwcmltYXJ5XCI+YnV0dG9uMjwvZWwtYnV0dG9uPlxuICAgIDxlbC1idXR0b24gdHlwZT1cInByaW1hcnlcIj5idXR0b24zIDEyMTI8L2VsLWJ1dHRvbj5cbiAgPC9lbC1idXR0b24tZ3JvdXA+XG4gIDxzcGFuPl9fXzwvc3Bhbj5cblxuICA8ZWwtYnV0dG9uLWdyb3VwPlxuICAgIDxlbC1idXR0b24gdHlwZT1cInByaW1hcnlcIj5idXR0b24xPC9lbC1idXR0b24+XG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwicHJpbWFyeVwiPmJ1dHRvbjI8L2VsLWJ1dHRvbj5cbiAgPC9lbC1idXR0b24tZ3JvdXA+XG4gIDxzcGFuPl9fXzwvc3Bhbj5cbiAgPGVsLWJ1dHRvbi1ncm91cD5cbiAgICA8ZWwtYnV0dG9uPmJ1dHRvbjE8L2VsLWJ1dHRvbj5cbiAgICA8ZWwtYnV0dG9uPmJ1dHRvbjI8L2VsLWJ1dHRvbj5cbiAgICA8ZWwtYnV0dG9uPmJ1dHRvbjMgMTIxMjwvZWwtYnV0dG9uPlxuICA8L2VsLWJ1dHRvbi1ncm91cD5cblxuICA8c3Bhbj5fX188L3NwYW4+XG5cbiAgPGVsLWJ1dHRvbi1ncm91cD5cbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJwcmltYXJ5XCIgcm91bmQ+cm91bmQ8L2VsLWJ1dHRvbj5cbiAgPC9lbC1idXR0b24tZ3JvdXA+XG4gIDxzcGFuPl9fXzwvc3Bhbj5cbiAgPGVsLWJ1dHRvbi1ncm91cD5cbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJwcmltYXJ5XCIgY2lyY2xlPmNpcmNsZTwvZWwtYnV0dG9uPlxuICA8L2VsLWJ1dHRvbi1ncm91cD5cbiAgPHNwYW4+X19fPC9zcGFuPlxuICA8ZWwtYnV0dG9uLWdyb3VwPlxuICAgIDxlbC1idXR0b24gdHlwZT1cInByaW1hcnlcIj5wcmltYXJ5PC9lbC1idXR0b24+XG4gIDwvZWwtYnV0dG9uLWdyb3VwPlxuICA8c3Bhbj5fX188L3NwYW4+XG4gIDxlbC1idXR0b24tZ3JvdXA+XG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwicHJpbWFyeVwiPmJ1dHRvbjMgMTIxMjwvZWwtYnV0dG9uPlxuXG4gICAgPGVsLWRyb3Bkb3duPlxuICAgICAgPGVsLWJ1dHRvbiB0eXBlPVwicHJpbWFyeVwiPlxuICAgICAgICBEcm9wZG93biBMaXN0PGVsLWljb24gY2xhc3M9XCJlbC1pY29uLS1yaWdodFwiPjxhcnJvdy1kb3duIC8+PC9lbC1pY29uPlxuICAgICAgPC9lbC1idXR0b24+XG4gICAgICA8dGVtcGxhdGUgI2Ryb3Bkb3duPlxuICAgICAgICA8ZWwtZHJvcGRvd24tbWVudT5cbiAgICAgICAgICA8ZWwtZHJvcGRvd24taXRlbT5BY3Rpb24gMTwvZWwtZHJvcGRvd24taXRlbT5cbiAgICAgICAgICA8ZWwtZHJvcGRvd24taXRlbT5BY3Rpb24gMjwvZWwtZHJvcGRvd24taXRlbT5cbiAgICAgICAgICA8ZWwtZHJvcGRvd24taXRlbT5BY3Rpb24gMzwvZWwtZHJvcGRvd24taXRlbT5cbiAgICAgICAgICA8ZWwtZHJvcGRvd24taXRlbT5BY3Rpb24gNDwvZWwtZHJvcGRvd24taXRlbT5cbiAgICAgICAgICA8ZWwtZHJvcGRvd24taXRlbT5BY3Rpb24gNTwvZWwtZHJvcGRvd24taXRlbT5cbiAgICAgICAgPC9lbC1kcm9wZG93bi1tZW51PlxuICAgICAgPC90ZW1wbGF0ZT5cbiAgICA8L2VsLWRyb3Bkb3duPlxuICA8L2VsLWJ1dHRvbi1ncm91cD5cblxuICA8YnIgLz5cbiAgPGJyIC8+XG5cbiAgPGVsLWJ1dHRvbi1ncm91cCBkaXJlY3Rpb249XCJ2ZXJ0aWNhbFwiPlxuICAgIDxlbC1idXR0b24+YnV0dG9uMTwvZWwtYnV0dG9uPlxuICAgIDxlbC1idXR0b24+YnV0dG9uMiAxPC9lbC1idXR0b24+XG4gICAgPGVsLWJ1dHRvbj5idXR0b24zIDEyMTI8L2VsLWJ1dHRvbj5cbiAgPC9lbC1idXR0b24tZ3JvdXA+XG4gIDxzcGFuPl9fXzwvc3Bhbj5cbiAgPGVsLWJ1dHRvbi1ncm91cCBkaXJlY3Rpb249XCJ2ZXJ0aWNhbFwiPlxuICAgIDxlbC1idXR0b24gdHlwZT1cInByaW1hcnlcIj5idXR0b24xPC9lbC1idXR0b24+XG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwicHJpbWFyeVwiPmJ1dHRvbjI8L2VsLWJ1dHRvbj5cbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJwcmltYXJ5XCI+YnV0dG9uMyAxMjEyPC9lbC1idXR0b24+XG4gIDwvZWwtYnV0dG9uLWdyb3VwPlxuICA8c3Bhbj5fX188L3NwYW4+XG4gIDxlbC1idXR0b24tZ3JvdXAgZGlyZWN0aW9uPVwidmVydGljYWxcIj5cbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJwcmltYXJ5XCIgcm91bmQ+cm91bmQ8L2VsLWJ1dHRvbj5cbiAgPC9lbC1idXR0b24tZ3JvdXA+XG4gIDxzcGFuPl9fXzwvc3Bhbj5cbiAgPGVsLWJ1dHRvbi1ncm91cCBkaXJlY3Rpb249XCJ2ZXJ0aWNhbFwiPlxuICAgIDxlbC1idXR0b24gdHlwZT1cInByaW1hcnlcIiBjaXJjbGU+Y2lyY2xlPC9lbC1idXR0b24+XG4gIDwvZWwtYnV0dG9uLWdyb3VwPlxuICA8c3Bhbj5fX188L3NwYW4+XG4gIDxlbC1idXR0b24tZ3JvdXAgZGlyZWN0aW9uPVwidmVydGljYWxcIj5cbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJwcmltYXJ5XCI+cHJpbWFyeTwvZWwtYnV0dG9uPlxuICA8L2VsLWJ1dHRvbi1ncm91cD5cbiAgPHNwYW4+X19fPC9zcGFuPlxuICA8ZWwtYnV0dG9uLWdyb3VwIGRpcmVjdGlvbj1cInZlcnRpY2FsXCI+XG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwicHJpbWFyeVwiPmJ1dHRvbjE8L2VsLWJ1dHRvbj5cbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJwcmltYXJ5XCI+YnV0dG9uMjwvZWwtYnV0dG9uPlxuICA8L2VsLWJ1dHRvbi1ncm91cD5cblxuICA8c3Bhbj5fX188L3NwYW4+XG4gIDxlbC1idXR0b24tZ3JvdXAgZGlyZWN0aW9uPVwidmVydGljYWxcIj5cbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJwcmltYXJ5XCI+YnV0dG9uMTwvZWwtYnV0dG9uPlxuICAgIDxlbC1kcm9wZG93bj5cbiAgICAgIDxlbC1idXR0b24gdHlwZT1cInByaW1hcnlcIj5cbiAgICAgICAgRHJvcGRvd24gTGlzdDxlbC1pY29uIGNsYXNzPVwiZWwtaWNvbi0tcmlnaHRcIj48YXJyb3ctZG93biAvPjwvZWwtaWNvbj5cbiAgICAgIDwvZWwtYnV0dG9uPlxuICAgICAgPHRlbXBsYXRlICNkcm9wZG93bj5cbiAgICAgICAgPGVsLWRyb3Bkb3duLW1lbnU+XG4gICAgICAgICAgPGVsLWRyb3Bkb3duLWl0ZW0+QWN0aW9uIDE8L2VsLWRyb3Bkb3duLWl0ZW0+XG4gICAgICAgICAgPGVsLWRyb3Bkb3duLWl0ZW0+QWN0aW9uIDI8L2VsLWRyb3Bkb3duLWl0ZW0+XG4gICAgICAgICAgPGVsLWRyb3Bkb3duLWl0ZW0+QWN0aW9uIDM8L2VsLWRyb3Bkb3duLWl0ZW0+XG4gICAgICAgICAgPGVsLWRyb3Bkb3duLWl0ZW0+QWN0aW9uIDQ8L2VsLWRyb3Bkb3duLWl0ZW0+XG4gICAgICAgICAgPGVsLWRyb3Bkb3duLWl0ZW0+QWN0aW9uIDU8L2VsLWRyb3Bkb3duLWl0ZW0+XG4gICAgICAgIDwvZWwtZHJvcGRvd24tbWVudT5cbiAgICAgIDwvdGVtcGxhdGU+XG4gICAgPC9lbC1kcm9wZG93bj5cbiAgPC9lbC1idXR0b24tZ3JvdXA+XG4gIDxiciAvPlxuICA8YnIgLz5cbjwvdGVtcGxhdGU+XG4iLCJlbGVtZW50LXBsdXMuanMiOiJpbXBvcnQgRWxlbWVudFBsdXMgZnJvbSAnZWxlbWVudC1wbHVzJ1xuaW1wb3J0IHsgZ2V0Q3VycmVudEluc3RhbmNlIH0gZnJvbSAndnVlJ1xuXG5sZXQgaW5zdGFsbGVkID0gZmFsc2VcbmF3YWl0IGxvYWRTdHlsZSgpXG5cbmV4cG9ydCBmdW5jdGlvbiBzZXR1cEVsZW1lbnRQbHVzKCkge1xuICBpZiAoaW5zdGFsbGVkKSByZXR1cm5cbiAgY29uc3QgaW5zdGFuY2UgPSBnZXRDdXJyZW50SW5zdGFuY2UoKVxuICBpbnN0YW5jZS5hcHBDb250ZXh0LmFwcC51c2UoRWxlbWVudFBsdXMpXG4gIGluc3RhbGxlZCA9IHRydWVcbn1cblxuZXhwb3J0IGZ1bmN0aW9uIGxvYWRTdHlsZSgpIHtcbiAgY29uc3Qgc3R5bGVzID0gWydodHRwczovL3ByZXZpZXctMTg5MDYtZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5jc3MnLCAnaHR0cHM6Ly9wcmV2aWV3LTE4OTA2LWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguY3NzJ10ubWFwKChzdHlsZSkgPT4ge1xuICAgIHJldHVybiBuZXcgUHJvbWlzZSgocmVzb2x2ZSwgcmVqZWN0KSA9PiB7XG4gICAgICBjb25zdCBsaW5rID0gZG9jdW1lbnQuY3JlYXRlRWxlbWVudCgnbGluaycpXG4gICAgICBsaW5rLnJlbCA9ICdzdHlsZXNoZWV0J1xuICAgICAgbGluay5ocmVmID0gc3R5bGVcbiAgICAgIGxpbmsuYWRkRXZlbnRMaXN0ZW5lcignbG9hZCcsIHJlc29sdmUpXG4gICAgICBsaW5rLmFkZEV2ZW50TGlzdGVuZXIoJ2Vycm9yJywgcmVqZWN0KVxuICAgICAgZG9jdW1lbnQuYm9keS5hcHBlbmQobGluaylcbiAgICB9KVxuICB9KVxuICByZXR1cm4gUHJvbWlzZS5hbGxTZXR0bGVkKHN0eWxlcylcbn1cbiIsInRzY29uZmlnLmpzb24iOiJ7XG4gIFwiY29tcGlsZXJPcHRpb25zXCI6IHtcbiAgICBcInRhcmdldFwiOiBcIkVTTmV4dFwiLFxuICAgIFwianN4XCI6IFwicHJlc2VydmVcIixcbiAgICBcIm1vZHVsZVwiOiBcIkVTTmV4dFwiLFxuICAgIFwibW9kdWxlUmVzb2x1dGlvblwiOiBcIkJ1bmRsZXJcIixcbiAgICBcInR5cGVzXCI6IFtcImVsZW1lbnQtcGx1cy9nbG9iYWwuZC50c1wiXSxcbiAgICBcImFsbG93SW1wb3J0aW5nVHNFeHRlbnNpb25zXCI6IHRydWUsXG4gICAgXCJhbGxvd0pzXCI6IHRydWUsXG4gICAgXCJjaGVja0pzXCI6IHRydWVcbiAgfSxcbiAgXCJ2dWVDb21waWxlck9wdGlvbnNcIjoge1xuICAgIFwidGFyZ2V0XCI6IDMuM1xuICB9XG59XG4iLCJQbGF5Z3JvdW5kTWFpbi52dWUiOiI8c2NyaXB0IHNldHVwPlxuaW1wb3J0IEFwcCBmcm9tICcuL0FwcC52dWUnXG5pbXBvcnQgeyBzZXR1cEVsZW1lbnRQbHVzIH0gZnJvbSAnLi9lbGVtZW50LXBsdXMuanMnXG5zZXR1cEVsZW1lbnRQbHVzKClcbjwvc2NyaXB0PlxuXG48dGVtcGxhdGU+XG4gIDxBcHAgLz5cbjwvdGVtcGxhdGU+XG4iLCJpbXBvcnQtbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7XG4gICAgXCJ2dWVcIjogXCJodHRwczovL2Nkbi5qc2RlbGl2ci5uZXQvbnBtL0B2dWUvcnVudGltZS1kb21AbGF0ZXN0L2Rpc3QvcnVudGltZS1kb20uZXNtLWJyb3dzZXIuanNcIixcbiAgICBcIkB2dWUvc2hhcmVkXCI6IFwiaHR0cHM6Ly9jZG4uanNkZWxpdnIubmV0L25wbS9AdnVlL3NoYXJlZEBsYXRlc3QvZGlzdC9zaGFyZWQuZXNtLWJ1bmRsZXIuanNcIixcbiAgICBcImVsZW1lbnQtcGx1c1wiOiBcImh0dHBzOi8vcHJldmlldy0xODkwNi1lbGVtZW50LXBsdXMuc3VyZ2Uuc2gvYnVuZGxlL2luZGV4LmZ1bGwubWluLm1qc1wiLFxuICAgIFwiZWxlbWVudC1wbHVzL1wiOiBcInVuc3VwcG9ydGVkXCIsXG4gICAgXCJAZWxlbWVudC1wbHVzL2ljb25zLXZ1ZVwiOiBcImh0dHBzOi8vY2RuLmpzZGVsaXZyLm5ldC9ucG0vQGVsZW1lbnQtcGx1cy9pY29ucy12dWVAMi9kaXN0L2luZGV4Lm1pbi5qc1wiXG4gIH0sXG4gIFwic2NvcGVzXCI6IHt9XG59IiwiX28iOnsic2hvd0hpZGRlbiI6dHJ1ZSwic3R5bGVTb3VyY2UiOiJodHRwczovL3ByZXZpZXctMTg5MDYtZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5jc3MifX0=)

VisualYuki committed:
Merge branch 'dev' into vertical-button-group

thinkasany commented:
> @thinkasany [updated example](https://element-plus.run/#eyJBcHAudnVlIjoiPHNjcmlwdCBzZXR1cCBsYW5nPVwidHNcIj5cbmltcG9ydCB7IEVsZW1lbnRQbHVzLCBBcnJvd0Rvd24gfSBmcm9tICdAZWxlbWVudC1wbHVzL2ljb25zLXZ1ZSdcbmltcG9ydCB7IHZlcnNpb24gYXMgZXBWZXJzaW9uIH0gZnJvbSAnZWxlbWVudC1wbHVzJ1xuaW1wb3J0IHsgcmVmLCB2ZXJzaW9uIGFzIHZ1ZVZlcnNpb24gfSBmcm9tICd2dWUnXG5cbmNvbnN0IG1zZyA9IHJlZignSGVsbG8gV29ybGQhJylcbjwvc2NyaXB0PlxuXG48dGVtcGxhdGU+XG4gIDxlbC1idXR0b24tZ3JvdXA+XG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwicHJpbWFyeVwiPmJ1dHRvbjE8L2VsLWJ1dHRvbj5cbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJwcmltYXJ5XCI+YnV0dG9uMjwvZWwtYnV0dG9uPlxuICAgIDxlbC1idXR0b24gdHlwZT1cInByaW1hcnlcIj5idXR0b24zIDEyMTI8L2VsLWJ1dHRvbj5cbiAgPC9lbC1idXR0b24tZ3JvdXA+XG4gIDxzcGFuPl9fXzwvc3Bhbj5cblxuICA8ZWwtYnV0dG9uLWdyb3VwPlxuICAgIDxlbC1idXR0b24gdHlwZT1cInByaW1hcnlcIj5idXR0b24xPC9lbC1idXR0b24+XG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwicHJpbWFyeVwiPmJ1dHRvbjI8L2VsLWJ1dHRvbj5cbiAgPC9lbC1idXR0b24tZ3JvdXA+XG4gIDxzcGFuPl9fXzwvc3Bhbj5cbiAgPGVsLWJ1dHRvbi1ncm91cD5cbiAgICA8ZWwtYnV0dG9uPmJ1dHRvbjE8L2VsLWJ1dHRvbj5cbiAgICA8ZWwtYnV0dG9uPmJ1dHRvbjI8L2VsLWJ1dHRvbj5cbiAgICA8ZWwtYnV0dG9uPmJ1dHRvbjMgMTIxMjwvZWwtYnV0dG9uPlxuICA8L2VsLWJ1dHRvbi1ncm91cD5cblxuICA8c3Bhbj5fX188L3NwYW4+XG5cbiAgPGVsLWJ1dHRvbi1ncm91cD5cbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJwcmltYXJ5XCIgcm91bmQ+cm91bmQ8L2VsLWJ1dHRvbj5cbiAgPC9lbC1idXR0b24tZ3JvdXA+XG4gIDxzcGFuPl9fXzwvc3Bhbj5cbiAgPGVsLWJ1dHRvbi1ncm91cD5cbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJwcmltYXJ5XCIgY2lyY2xlPmNpcmNsZTwvZWwtYnV0dG9uPlxuICA8L2VsLWJ1dHRvbi1ncm91cD5cbiAgPHNwYW4+X19fPC9zcGFuPlxuICA8ZWwtYnV0dG9uLWdyb3VwPlxuICAgIDxlbC1idXR0b24gdHlwZT1cInByaW1hcnlcIj5wcmltYXJ5PC9lbC1idXR0b24+XG4gIDwvZWwtYnV0dG9uLWdyb3VwPlxuICA8c3Bhbj5fX188L3NwYW4+XG4gIDxlbC1idXR0b24tZ3JvdXA+XG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwicHJpbWFyeVwiPmJ1dHRvbjMgMTIxMjwvZWwtYnV0dG9uPlxuXG4gICAgPGVsLWRyb3Bkb3duPlxuICAgICAgPGVsLWJ1dHRvbiB0eXBlPVwicHJpbWFyeVwiPlxuICAgICAgICBEcm9wZG93biBMaXN0PGVsLWljb24gY2xhc3M9XCJlbC1pY29uLS1yaWdodFwiPjxhcnJvdy1kb3duIC8+PC9lbC1pY29uPlxuICAgICAgPC9lbC1idXR0b24+XG4gICAgICA8dGVtcGxhdGUgI2Ryb3Bkb3duPlxuICAgICAgICA8ZWwtZHJvcGRvd24tbWVudT5cbiAgICAgICAgICA8ZWwtZHJvcGRvd24taXRlbT5BY3Rpb24gMTwvZWwtZHJvcGRvd24taXRlbT5cbiAgICAgICAgICA8ZWwtZHJvcGRvd24taXRlbT5BY3Rpb24gMjwvZWwtZHJvcGRvd24taXRlbT5cbiAgICAgICAgICA8ZWwtZHJvcGRvd24taXRlbT5BY3Rpb24gMzwvZWwtZHJvcGRvd24taXRlbT5cbiAgICAgICAgICA8ZWwtZHJvcGRvd24taXRlbT5BY3Rpb24gNDwvZWwtZHJvcGRvd24taXRlbT5cbiAgICAgICAgICA8ZWwtZHJvcGRvd24taXRlbT5BY3Rpb24gNTwvZWwtZHJvcGRvd24taXRlbT5cbiAgICAgICAgPC9lbC1kcm9wZG93bi1tZW51PlxuICAgICAgPC90ZW1wbGF0ZT5cbiAgICA8L2VsLWRyb3Bkb3duPlxuICA8L2VsLWJ1dHRvbi1ncm91cD5cblxuICA8YnIgLz5cbiAgPGJyIC8+XG5cbiAgPGVsLWJ1dHRvbi1ncm91cCBkaXJlY3Rpb249XCJ2ZXJ0aWNhbFwiPlxuICAgIDxlbC1idXR0b24+YnV0dG9uMTwvZWwtYnV0dG9uPlxuICAgIDxlbC1idXR0b24+YnV0dG9uMiAxPC9lbC1idXR0b24+XG4gICAgPGVsLWJ1dHRvbj5idXR0b24zIDEyMTI8L2VsLWJ1dHRvbj5cbiAgPC9lbC1idXR0b24tZ3JvdXA+XG4gIDxzcGFuPl9fXzwvc3Bhbj5cbiAgPGVsLWJ1dHRvbi1ncm91cCBkaXJlY3Rpb249XCJ2ZXJ0aWNhbFwiPlxuICAgIDxlbC1idXR0b24gdHlwZT1cInByaW1hcnlcIj5idXR0b24xPC9lbC1idXR0b24+XG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwicHJpbWFyeVwiPmJ1dHRvbjI8L2VsLWJ1dHRvbj5cbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJwcmltYXJ5XCI+YnV0dG9uMyAxMjEyPC9lbC1idXR0b24+XG4gIDwvZWwtYnV0dG9uLWdyb3VwPlxuICA8c3Bhbj5fX188L3NwYW4+XG4gIDxlbC1idXR0b24tZ3JvdXAgZGlyZWN0aW9uPVwidmVydGljYWxcIj5cbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJwcmltYXJ5XCIgcm91bmQ+cm91bmQ8L2VsLWJ1dHRvbj5cbiAgPC9lbC1idXR0b24tZ3JvdXA+XG4gIDxzcGFuPl9fXzwvc3Bhbj5cbiAgPGVsLWJ1dHRvbi1ncm91cCBkaXJlY3Rpb249XCJ2ZXJ0aWNhbFwiPlxuICAgIDxlbC1idXR0b24gdHlwZT1cInByaW1hcnlcIiBjaXJjbGU+Y2lyY2xlPC9lbC1idXR0b24+XG4gIDwvZWwtYnV0dG9uLWdyb3VwPlxuICA8c3Bhbj5fX188L3NwYW4+XG4gIDxlbC1idXR0b24tZ3JvdXAgZGlyZWN0aW9uPVwidmVydGljYWxcIj5cbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJwcmltYXJ5XCI+cHJpbWFyeTwvZWwtYnV0dG9uPlxuICA8L2VsLWJ1dHRvbi1ncm91cD5cbiAgPHNwYW4+X19fPC9zcGFuPlxuICA8ZWwtYnV0dG9uLWdyb3VwIGRpcmVjdGlvbj1cInZlcnRpY2FsXCI+XG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwicHJpbWFyeVwiPmJ1dHRvbjE8L2VsLWJ1dHRvbj5cbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJwcmltYXJ5XCI+YnV0dG9uMjwvZWwtYnV0dG9uPlxuICA8L2VsLWJ1dHRvbi1ncm91cD5cblxuICA8c3Bhbj5fX188L3NwYW4+XG4gIDxlbC1idXR0b24tZ3JvdXAgZGlyZWN0aW9uPVwidmVydGljYWxcIj5cbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJwcmltYXJ5XCI+YnV0dG9uMTwvZWwtYnV0dG9uPlxuICAgIDxlbC1kcm9wZG93bj5cbiAgICAgIDxlbC1idXR0b24gdHlwZT1cInByaW1hcnlcIj5cbiAgICAgICAgRHJvcGRvd24gTGlzdDxlbC1pY29uIGNsYXNzPVwiZWwtaWNvbi0tcmlnaHRcIj48YXJyb3ctZG93biAvPjwvZWwtaWNvbj5cbiAgICAgIDwvZWwtYnV0dG9uPlxuICAgICAgPHRlbXBsYXRlICNkcm9wZG93bj5cbiAgICAgICAgPGVsLWRyb3Bkb3duLW1lbnU+XG4gICAgICAgICAgPGVsLWRyb3Bkb3duLWl0ZW0+QWN0aW9uIDE8L2VsLWRyb3Bkb3duLWl0ZW0+XG4gICAgICAgICAgPGVsLWRyb3Bkb3duLWl0ZW0+QWN0aW9uIDI8L2VsLWRyb3Bkb3duLWl0ZW0+XG4gICAgICAgICAgPGVsLWRyb3Bkb3duLWl0ZW0+QWN0aW9uIDM8L2VsLWRyb3Bkb3duLWl0ZW0+XG4gICAgICAgICAgPGVsLWRyb3Bkb3duLWl0ZW0+QWN0aW9uIDQ8L2VsLWRyb3Bkb3duLWl0ZW0+XG4gICAgICAgICAgPGVsLWRyb3Bkb3duLWl0ZW0+QWN0aW9uIDU8L2VsLWRyb3Bkb3duLWl0ZW0+XG4gICAgICAgIDwvZWwtZHJvcGRvd24tbWVudT5cbiAgICAgIDwvdGVtcGxhdGU+XG4gICAgPC9lbC1kcm9wZG93bj5cbiAgPC9lbC1idXR0b24tZ3JvdXA+XG4gIDxiciAvPlxuICA8YnIgLz5cbjwvdGVtcGxhdGU+XG4iLCJlbGVtZW50LXBsdXMuanMiOiJpbXBvcnQgRWxlbWVudFBsdXMgZnJvbSAnZWxlbWVudC1wbHVzJ1xuaW1wb3J0IHsgZ2V0Q3VycmVudEluc3RhbmNlIH0gZnJvbSAndnVlJ1xuXG5sZXQgaW5zdGFsbGVkID0gZmFsc2VcbmF3YWl0IGxvYWRTdHlsZSgpXG5cbmV4cG9ydCBmdW5jdGlvbiBzZXR1cEVsZW1lbnRQbHVzKCkge1xuICBpZiAoaW5zdGFsbGVkKSByZXR1cm5cbiAgY29uc3QgaW5zdGFuY2UgPSBnZXRDdXJyZW50SW5zdGFuY2UoKVxuICBpbnN0YW5jZS5hcHBDb250ZXh0LmFwcC51c2UoRWxlbWVudFBsdXMpXG4gIGluc3RhbGxlZCA9IHRydWVcbn1cblxuZXhwb3J0IGZ1bmN0aW9uIGxvYWRTdHlsZSgpIHtcbiAgY29uc3Qgc3R5bGVzID0gWydodHRwczovL3ByZXZpZXctMTg5MDYtZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5jc3MnLCAnaHR0cHM6Ly9wcmV2aWV3LTE4OTA2LWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguY3NzJ10ubWFwKChzdHlsZSkgPT4ge1xuICAgIHJldHVybiBuZXcgUHJvbWlzZSgocmVzb2x2ZSwgcmVqZWN0KSA9PiB7XG4gICAgICBjb25zdCBsaW5rID0gZG9jdW1lbnQuY3JlYXRlRWxlbWVudCgnbGluaycpXG4gICAgICBsaW5rLnJlbCA9ICdzdHlsZXNoZWV0J1xuICAgICAgbGluay5ocmVmID0gc3R5bGVcbiAgICAgIGxpbmsuYWRkRXZlbnRMaXN0ZW5lcignbG9hZCcsIHJlc29sdmUpXG4gICAgICBsaW5rLmFkZEV2ZW50TGlzdGVuZXIoJ2Vycm9yJywgcmVqZWN0KVxuICAgICAgZG9jdW1lbnQuYm9keS5hcHBlbmQobGluaylcbiAgICB9KVxuICB9KVxuICByZXR1cm4gUHJvbWlzZS5hbGxTZXR0bGVkKHN0eWxlcylcbn1cbiIsInRzY29uZmlnLmpzb24iOiJ7XG4gIFwiY29tcGlsZXJPcHRpb25zXCI6IHtcbiAgICBcInRhcmdldFwiOiBcIkVTTmV4dFwiLFxuICAgIFwianN4XCI6IFwicHJlc2VydmVcIixcbiAgICBcIm1vZHVsZVwiOiBcIkVTTmV4dFwiLFxuICAgIFwibW9kdWxlUmVzb2x1dGlvblwiOiBcIkJ1bmRsZXJcIixcbiAgICBcInR5cGVzXCI6IFtcImVsZW1lbnQtcGx1cy9nbG9iYWwuZC50c1wiXSxcbiAgICBcImFsbG93SW1wb3J0aW5nVHNFeHRlbnNpb25zXCI6IHRydWUsXG4gICAgXCJhbGxvd0pzXCI6IHRydWUsXG4gICAgXCJjaGVja0pzXCI6IHRydWVcbiAgfSxcbiAgXCJ2dWVDb21waWxlck9wdGlvbnNcIjoge1xuICAgIFwidGFyZ2V0XCI6IDMuM1xuICB9XG59XG4iLCJQbGF5Z3JvdW5kTWFpbi52dWUiOiI8c2NyaXB0IHNldHVwPlxuaW1wb3J0IEFwcCBmcm9tICcuL0FwcC52dWUnXG5pbXBvcnQgeyBzZXR1cEVsZW1lbnRQbHVzIH0gZnJvbSAnLi9lbGVtZW50LXBsdXMuanMnXG5zZXR1cEVsZW1lbnRQbHVzKClcbjwvc2NyaXB0PlxuXG48dGVtcGxhdGU+XG4gIDxBcHAgLz5cbjwvdGVtcGxhdGU+XG4iLCJpbXBvcnQtbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7XG4gICAgXCJ2dWVcIjogXCJodHRwczovL2Nkbi5qc2RlbGl2ci5uZXQvbnBtL0B2dWUvcnVudGltZS1kb21AbGF0ZXN0L2Rpc3QvcnVudGltZS1kb20uZXNtLWJyb3dzZXIuanNcIixcbiAgICBcIkB2dWUvc2hhcmVkXCI6IFwiaHR0cHM6Ly9jZG4uanNkZWxpdnIubmV0L25wbS9AdnVlL3NoYXJlZEBsYXRlc3QvZGlzdC9zaGFyZWQuZXNtLWJ1bmRsZXIuanNcIixcbiAgICBcImVsZW1lbnQtcGx1c1wiOiBcImh0dHBzOi8vcHJldmlldy0xODkwNi1lbGVtZW50LXBsdXMuc3VyZ2Uuc2gvYnVuZGxlL2luZGV4LmZ1bGwubWluLm1qc1wiLFxuICAgIFwiZWxlbWVudC1wbHVzL1wiOiBcInVuc3VwcG9ydGVkXCIsXG4gICAgXCJAZWxlbWVudC1wbHVzL2ljb25zLXZ1ZVwiOiBcImh0dHBzOi8vY2RuLmpzZGVsaXZyLm5ldC9ucG0vQGVsZW1lbnQtcGx1cy9pY29ucy12dWVAMi9kaXN0L2luZGV4Lm1pbi5qc1wiXG4gIH0sXG4gIFwic2NvcGVzXCI6IHt9XG59IiwiX28iOnsic2hvd0hpZGRlbiI6dHJ1ZSwic3R5bGVTb3VyY2UiOiJodHRwczovL3ByZXZpZXctMTg5MDYtZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5jc3MifX0=)

I'm not a core member, you can request a review from @btea , their suggestions are more useful.I have submitted similar functions before, I hope it will be helpful to you. #18653

VisualYuki committed:
feat(components): [button-group]: fix docs

VisualYuki committed:
Merge branch 'vertical-button-group' of https://github.com/VisualYuki/fork-element-plus into vertical-button-group

VisualYuki committed:
feat(components): [button-group]: update version

VisualYuki committed:
Merge branch 'element-plus:dev' into vertical-button-group

VisualYuki committed:
Merge branch 'dev' into vertical-button-group

VisualYuki committed:
Merge branch 'dev' into vertical-button-group

VisualYuki committed:
Merge branch 'dev' into vertical-button-group

btea commented on docs/en-US/component/button.md:
```diff
@@ -180,10 +182,11 @@ button/custom
+| ------------------ | ------------------------------------------------ | ------------------------------------------------------------------ | ---------- |
+| size               | control the size of buttons in this button-group | ^[enum]`'large' \| 'default' \| 'small'`                           | —          |
+| type               | control the type of buttons in this button-group | ^[enum]`'primary' \| 'success' \| 'warning' \| 'danger' \| 'info'` | —          |
+| direction ^(2.9.0) | display direction                                | ^[enum]`'horizontal' \| 'vertical'`                                | horizontal |
```
```suggestion
| direction ^(2.9.4) | display direction                                | ^[enum]`'horizontal' \| 'vertical'`                                | horizontal |
```

VisualYuki committed:
Update docs/en-US/component/button.md

Co-authored-by: btea <2356281422@qq.com>

tolking reviewed (approved):
LGTM.

btea committed:
Update docs/en-US/component/button.md

Dsaquel committed:
Merge branch 'dev' into vertical-button-group

Dsaquel committed:
chore: format

Dsaquel committed:
docs: improve sentence

Dsaquel committed:
docs: improve display example

Dsaquel committed:
refactor: enhance prop type

- fit with segmented direction prop
- enhance type according with
https://github.com/element-plus/element-plus/pull/22757
